This brings sgae 6.4 in line with 6.4.1 in terms of release.
diff --git a/src/sage/interfaces/maxima_abstract.py b/src/sage/interfaces/maxima_abstract.py
index 4ce4952..923ba9b 100644
--- a/src/sage/interfaces/maxima_abstract.py
+++ b/src/sage/interfaces/maxima_abstract.py
@@ -165,9 +165,9 @@ def _command_runner(self, command, s, redirect=True):
             p = subprocess.Popen(cmd, shell=True, stdin=subprocess.PIPE,
                              stdout=subprocess.PIPE, stderr=subprocess.PIPE)
             res = p.stdout.read()
-            # We now get 16 lines of commented verbosity
+            # We get 4 lines of commented verbosity
             # every time Maxima starts, so we need to get rid of them
-            for _ in range(16):
+            for _ in range(4):
                 res = res[res.find('\n')+1:]
             return AsciiArtString(res)
         else:
diff --git a/src/sage/interfaces/maxima_lib.py b/src/sage/interfaces/maxima_lib.py
index 681893f..de2a583 100644
--- a/src/sage/interfaces/maxima_lib.py
+++ b/src/sage/interfaces/maxima_lib.py
@@ -76,7 +76,6 @@
 
 ## We begin here by initializing Maxima in library mode
 ## i.e. loading it into ECL
-ecl_eval("(setf *compile-verbose* NIL)")
 ecl_eval("(setf *load-verbose* NIL)")
 ecl_eval("(require 'maxima)")
 ecl_eval("(in-package :maxima)")
